src/app/core/site-header/site-header.component.ts

```diff
@@ -43,6 +43,18 @@ export class SiteHeaderComponent {
     });
   }
 
+  @ViewChild('navbarSupportedContent', { read: ElementRef }) navBar!: ElementRef<HTMLElement>;
+
+  ngAfterViewInit(): void {
+    this.navBar.nativeElement.addEventListener('shown.bs.collapse', () => {
+      this.isOpen = true;
+    });
+
+    this.navBar.nativeElement.addEventListener('hidden.bs.collapse', () => {
```

Comment:
należy się odpinać od eventListenerów.
Mozna to zrobić na dwa sposoby
1. Użycie Renderer2
const functionToRemoveEventListener = this.render.listen(<element>, 'shown.bs.collapse', () => {});
w ngOnDestroy -> functionToRemoveEventListener();

2. fromEvent z rxjs fromEvent(this.navBar.nativeElement, shown.bs.collapse).subscribe()
a) użyc .pipe(takeUntil(this.eventSub$))